meta-resin-common/recipes-core/systemd/systemd_%.bbappend

```diff
@@ -10,6 +10,17 @@ SRC_URI_append = " \
     file://resin_update_state_probe \
     "
 
+python() {
+	pv = d.getVar('PV', True)
+	srcuri = d.getVar('SRC_URI', True)
+
+	# Versions before 236 are affected by CVE-2017-15908
+	# https://nvd.nist.gov/vuln/detail/CVE-2017-15908
+	# Backport the relevant patch on the affected versions
+	if int(pv) < 236:
```

Comment:
Failed: https://jenkins.dev.resin.io/job/yocto-odroid-c1/836/console

File: '__anon_22__yocto_resin_board_build____layers_meta_resin_meta_resin_common_recipes_core_systemd_systemd___bbappend', lineno: 8, function: __anon_22__yocto_resin_board_build____layers_meta_resin_meta_resin_common_recipes_core_systemd_systemd___bbappend
     0004:
     0005:	# Versions before 236 are affected by CVE-2017-15908
     0006:	# https://nvd.nist.gov/vuln/detail/CVE-2017-15908
     0007:	# Backport the relevant patch on the affected versions
 *** 0008:	if int(pv) < 236:
     0009:		d.setVar('SRC_URI', srcuri + ' file://0001-resolved-fix-loop-on-packets-with-pseudo-dns-types.patch')
     0010:
Exception: ValueError: invalid literal for int() with base 10: '225+gitAUTOINC+e1439a1472'

ERROR: Failed to parse recipe: /yocto/resin-board/build/../layers/poky/meta/recipes-core/systemd/systemd_225.bb